go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/golang/protobuf v1.5.2
 	github.com/gorilla/mux v1.8.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
+	github.com/osmosis-labs/osmosis v1.0.4
```

Comment:
We missed an import somewhere right? I don't think we should be importing v1 anywhere

Reply:
definitely.  I should have marked this one as draft.  

v3 is ready to rock.